code/game/machinery/doors/airlock.dm

```diff
@@ -314,16 +314,17 @@
 	M.forceMove(loc)
 
 /obj/machinery/door/airlock/attack_hand(mob/user)
+	if(isElectrified() && !issilicon(user))
+		shock(user, 100)
 	. = ..()
 	if(.)
 		return
-	if(!issilicon(user) && isElectrified())
-		shock(user, 100)
+
+
 
 /obj/machinery/door/airlock/Topic(href, href_list, var/nowindow = 0)
-	. = ..()
-	if(.)
-		return
+	if(!nowindow)
+		..()
```

Comment:
Are you aware of what this change and removing parent logic means?

Reply:
Likely due to merging not done correctly my end.